docs/blog/posts/2023/single-tier-dc.md

```diff
@@ -0,0 +1,153 @@
+
+## The 1-Rack Solution
+
+You can spin a Datacenter with as little as just two Top-of-Rack (ToRs) from [7220 IXR-D](https://www.nokia.com/networks/data-center/data-center-fabric/7220-interconnect-router/) family devices sitting on top of your rack of servers. The ToR switches are connected with their uplinks to an existing core for North-South traffic.
+
+![pic1](https://gitlab.com/rdodin/pics/-/wikis/uploads/3efa581a5c39d721a1a8d79e48ca26b2/image__1_.webp){.img-shadow}
+
+### Components
+
+**Servers**
+
+* Multi-home the servers to the two ToRs.
+
+* Create LACP (or static) LAG interfaces with different links ending on different ToRs to act as uplink from each server.
+
+**ToRs**
+
+We use Top-of-Rack and Leaf terms interchangeably in this post, both terms indicate a switch that servers in a rack are connected.
+
+* Provision one or two IP interfaces between the ToRs.
+
+* Use these interfaces as underlay for iBGP peering needed for EVPN-MH. Connectivity to the core provides backup underlay reachability to iBGP peer. This underlay can be established via eBGP or IGP.
+
+* [Enable All-Active EVPN-MH](https://documentation.nokia.com/srlinux/23-3/books/advanced-solutions/evpn-vxlan-layer-2-multi-hom.html#all-active-multi-hom-configurations) on the Leaf.
+
+* Enable Integrated Routing and Bridging (IRB) with anycast functionality on ToRs if L3 services are needed.
+
+EVPN-MH procedures will set up the data path to ensure a loop-free, fully redundant and load balanced layer 2 and layer 3 topology.
+
+The various failure scenarios that pose big challenges to traditional MC-LAG setups are seamlessly handled as explained in the next section.
+
+### Handling Network Failure
+
+#### Leaf-Server Link Failure
+
+When the link connecting Leaf to one of the servers fails, LACP detects this event at the server. This allows the server to react locally by dropping the failed link from its LAG membership and utilizing the alternate uplink.
+
+<figure markdown>
+  ![pic2](https://gitlab.com/rdodin/pics/-/wikis/uploads/85181b3dfcc942b5a28f2782baae874c/image__2_.webp){ .img-shadow width="400" }
+</figure>
+
+For datacenters requiring more levels of resiliency, there are two options:
+
+* Provision two links between the server and each Leaf: This provides link level resiliency in addition to node (Leaf) level resiliency.
+
+* Connect each server to up to 4 Leaf switches.
+
+#### Inter-Chassis Link Failure
+
+<figure markdown>
+  ![pic2](https://gitlab.com/rdodin/pics/-/wikis/uploads/e866147daba870b0db2c24bef27d82c6/image__3_.webp){ .img-shadow width="400" }
+</figure>
+
+The inter-chassis link should be formed using multiple IP interfaces. This allows one level of failure protection. However, if this all these links fail (say due to a fiber cut), a Leaf still has two alternate paths to reach its iBGP peer via Core. This provides multiple levels of protection against what is called Split-Brain situation.
```

Comment:
pending: consider removing ICL from EVPN MH deployment scenario as typically it is not used

Reply:
Jorge also mentioned that it is not required to mention ICL when we talk about EVPN-MH in all-active mode.

Reply:
discard this
we have to discuss this more offline